tasks/main.yml

```diff
@@ -15,32 +15,61 @@
     repo: '{{ docker__upstream_repository }}'
     state: 'present'
     update_cache: True
+  register: docker__register_apt_repo
   when: docker__upstream|d() | bool
 
+- name: Make sure that systemd directory exists
+  file:
+    path: '/etc/systemd/system'
+    state: 'directory'
+    owner: 'root'
+    group: 'root'
+    mode: '0755'
+  when: ansible_service_mgr == 'systemd'
+  tags: [ 'role::docker:config' ]
```

Comment:
Since the task below creates a subdirectory of this directory, this task most likely is not needed.